vms/event/views.py

```
@@ -48,17 +49,43 @@ class EventCreateView(LoginRequiredMixin, AdministratorLoginRequiredMixin,
     template_name = 'event/create.html'
     form_class = EventForm
 
+    def get_context_data(self, **kwargs):
+        context = super(EventCreateView, self).get_context_data(**kwargs)
+        context['country_list'] = Country.objects.all()
+        return context
+
     def form_valid(self, form):
+        country_list = Country.objects.all()
         start_date = form.cleaned_data['start_date']
         if start_date < (datetime.date.today() - datetime.timedelta(days=1)):
             messages.add_message(
                 self.request, messages.INFO,
                 'Start date should be today\'s date or later.')
             return render(self.request, 'event/create.html', {
                 'form': form,
+                'country_list': country_list,
             })
         else:
-            form.save()
+            event = form.save(commit=False)
+            try:
+                country_id = self.request.POST.get('country')
+                country = Country.objects.get(pk=country_id)
+                event.country = country
+            except:
```

bare except should never be used. Use at least `Exception`